src/app/Topology/Shared/utils.tsx

```diff
@@ -230,8 +232,8 @@ export const isTargetNodeFiltered = ({ target }: TargetNode, filters?: TopologyF
 export class SearchExprService {
   private readonly _state$ = new BehaviorSubject<string>('');
 
-  searchExpression(): Observable<string> {
-    return this._state$.asObservable();
+  searchExpression(debounceMs = 0): Observable<string> {
```

Comment:
I think it'd be better to use the `DEFAULT_MATCH_EXPR_DEBOUNCE_TIME` as the, well, default parameter value here. Seems like it would be more expected as a developer/maintainer coming along down the line and calling this function.